pipelines/smartseq2_single_sample/README.md

```diff
@@ -1,254 +1,184 @@
+  * [Quick Start Table](#quick-start-table)
+- [Set-Up](#set-up)
+  * [Smartseq2 Installation and Requirements](#smartseq2-installation-and-requirements)
+  * [Inputs](#inputs)
+    + [Sample data input](#sample-data-input)
+    + [Additional Reference Inputs:](#additional-reference-inputs-)
+- [Running Smartseq2](#running-smartseq2)
+  * [Smartsseq2 Task Summary](#smartsseq2-task-summary)
+    + [Part 1: Quality Control Tasks](#part-1--quality-control-tasks)
+      - [1.1 Align reads to the genome using HISAT2](#11-align-reads-to-the-genome-using-hisat2)
+      - [1.2 Calculate summary metrics using Picard](#12-calculate-summary-metrics-using-picard)
+    + [Part 2: Transcriptome Quantification Tasks](#part-2-transcriptome-quantification-tasks)
+      - [2.1 Align reads to the transcriptome using HISAT2](#21-align-reads-to-the-transcriptome-using-hisat2)
+      - [2.2 Quantify gene expression using RSEM](#22-quantify-gene-expression-using-rsem)
+  * [Outputs](#outputs)
+- [Versioning](#versioning)
+- [Improvements](#have-suggestions)
+
+# Smartseq2 Single Sample Pipeline Overview
+
+![](smartseq_image.png)
+
+## Introduction to the Smartseq2 Pipeline
+
+The Smartseq2 Single Sample pipeline is designed by the [Data Coordination Platform](https://data.humancellatlas.org/about) of the [Human Cell Atlas](https://www.humancellatlas.org/) to process single-cell RNAseq (scRNAseq) data generated by [Smartseq2 assays](https://www.nature.com/articles/nmeth.2639). The pipeline's workflow is written in WDL, is freely available on [GitHub](https://github.com/HumanCellAtlas/skylab), and can be run by any compliant WDL runner (e.g. [crowmell](https://github.com/broadinstitute/cromwell)). The pipeline is designed to process stranded or unstranded, paired- or single-end, scRNA-seq data from an individual cell (to process multiple cells, see the [smartseq2_multisample](https://github.com/HumanCellAtlas/skylab/tree/master/pipelines/smartseq2_multisample) pipeline). 
+
+The Smartseq2 Single Sample workflow is divided into two parts. First, it aligns reads to the genome and performs quality control on genome-aligned BAMs. Second, it aligns reads to the transcriptome and generates gene expression estimates from transcriptome-aligned BAMs. The pipeline returns reads and expression estimates in BAM and an optional Zarr format, and it returns QC metrics in CSV format. 
+
+This pipeline is validated to support both human and mouse datasets.   
+
+## Quick Start Table
+
+| Pipeline Features | Description | Source |
+|-------------------|---------------------------------------------------------------|-----------------------|
+| Assay Type | Smartseq2 Single Sample | [Smartseq2](https://www.nature.com/articles/nprot.2014.006)
+| Overall Workflow  | Quality control module and transcriptome quantification module | Code available from [Github](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl) |
```

Comment:
I suggest we remove the word module, there is no "module anywhere", just two parts to the pipeline as mentioned above. Whatever word we use to describe the two parts we should be consistent.